modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -85,6 +85,44 @@ RETURN a.name, a.year
 
 | Extension of the xref:clauses/load-csv.adoc#azure-cloud-storage[LOAD CSV] clause to allow loading CSV files from Azure Cloud Storage URIs.
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+CREATE USER bob
```

Comment:
should probably mention/show ALTER as well 🤔

Reply:
done, thanks